Source/WebCore/platform/mediastream/cocoa/DisplayCaptureSourceCocoa.h

```diff
@@ -152,6 +153,7 @@ class DisplayCaptureSourceCocoa final
     Seconds m_elapsedTime { 0_s };
 
     RunLoop::Timer<DisplayCaptureSourceCocoa> m_timer;
+    UserActivity m_userActivity;
```

Comment:
This is apparently not instantiated in the WebProcess so it is not redundant after all.

Reply:
Right, class is instantiated either in UIProcess or GPUProcess.